src/main/scala/com/danielasfregola/twitter4s/entities/DirectMessageEvent.scala

```diff
@@ -0,0 +1,18 @@
+package com.danielasfregola.twitter4s.entities
+
+final case class DirectMessageEvent(`type`: String,
```

Comment:
`type` should be an enum as we should know the set of possible values accepted by the Twitter API
`id`s general are of type Long, any particular reason they are of type String here?
`timestamp`s should have a type consistent to the other types of timestamp in the code. Also, we may want to add a default to "now", so that people can just create the entity without providing one explicitly

Reply:
done